e2e/test/scenarios/dashboard-filters/dashboard-chained-filters.cy.spec.js

```diff
@@ -141,71 +144,106 @@ describe("scenarios > dashboard > chained filter", () => {
+
+        cy.createQuestion({
+          name: "15170",
+          database: WRITABLE_DB_ID,
+          query: { "source-table": TEST_TABLE_ID },
+        }).then(({ body: { id: QUESTION_ID } }) => {
+          cy.createDashboard().then(({ body: { id: DASHBOARD_ID } }) => {
+            // Add filter to the dashboard
+            cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}`, {
+              parameters: [
+                {
+                  id: "50c9eac6",
+                  name: "ID",
+                  slug: "id",
+                  type: "id",
+                },
+              ],
+            });
 
-        // Add previously created question to the dashboard
-        addOrUpdateDashboardCard({
-          card_id: QUESTION_ID,
-          dashboard_id: DASHBOARD_ID,
-        }).then(({ body: { id: DASH_CARD_ID } }) => {
-          // Connect filter to that question
-          cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}/cards`, {
-            cards: [
-              {
-                id: DASH_CARD_ID,
-                card_id: QUESTION_ID,
-                row: 0,
-                col: 0,
-                size_x: 11,
-                size_y: 6,
-                parameter_mappings: [
+            // Add previously created question to the dashboard
+            addOrUpdateDashboardCard({
+              card_id: QUESTION_ID,
+              dashboard_id: DASHBOARD_ID,
+            }).then(({ body: { id: DASH_CARD_ID } }) => {
+              // Connect filter to that question
+              cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}/cards`, {
+                cards: [
                   {
-                    parameter_id: "50c9eac6",
+                    id: DASH_CARD_ID,
                     card_id: QUESTION_ID,
-                    target: ["dimension", ["field-id", PRODUCTS.EAN]],
+                    row: 0,
+                    col: 0,
+                    size_x: 11,
+                    size_y: 6,
+                    parameter_mappings: [
+                      {
+                        parameter_id: "50c9eac6",
+                        card_id: QUESTION_ID,
+                        target: ["dimension", ["field-id", UUID_FIELD_ID]],
+                      },
+                    ],
                   },
                 ],
-              },
-            ],
-          });
-        });
+              });
+            });
 
-        visitDashboard(DASHBOARD_ID);
-        cy.icon("pencil").click();
-        showDashboardCardActions();
-        cy.icon("click").click();
-        cy.findByText(/Ean/i).click();
-        cy.findByText("Update a dashboard filter").click();
-        cy.findByText("Available filters").parent().findByText(/ID/i).click();
-        popover().within(() => {
-          cy.findByText(/Ean/i);
+            visitDashboard(DASHBOARD_ID);
+            cy.icon("pencil").click();
+            showDashboardCardActions();
+            getDashboardCard().icon("click").click();
```

Comment:
When using the new writable database, Metabase detects that actions are enabled, so it shows the action icon in the dashboard header which has the same `click` icon. That's why I need to target `click` icon inside the dashcard instead.
![Screenshot 2023-08-17 at 1 44 55 PM](https://github.com/metabase/metabase/assets/1937582/154f8033-d2c4-4e92-9acc-ff6f8cab75d4)